app/controllers/register/beneficiaries/charityortrust/charity/CharityInternationalAddressController.scala

```diff
@@ -72,10 +76,15 @@ class CharityInternationalAddressController @Inject()(
           Future.successful(BadRequest(view(formWithErrors, countryOptions.options, draftId, index, charityName))),
 
         value => {
-          for {
-            updatedAnswers <- Future.fromTry(request.userAnswers.set(CharityInternationalAddressPage(index), value))
-            _              <- repository.set(updatedAnswers)
+          val result = for {
```

Comment:
It doesn't look like there's test coverage for this controller. You could add some tests to confirm that the redirects go to the correct place

Reply:
I think this would be better done as a seperate ticket to look at improving the overall test quality

Reply:
sure